nvme.c

```diff
@@ -5317,6 +5321,330 @@ static int show_registers(int argc, char **argv, struct command *cmd, struct plu
 	return err;
 }
 
+bool nvme_is_fabrics_reg(int offset)
+{
+	switch (offset) {
+	case NVME_REG_CAP:
+		fallthrough;
+	case NVME_REG_VS:
+		fallthrough;
+	case NVME_REG_CC:
+		fallthrough;
+	case NVME_REG_CSTS:
+		fallthrough;
+	case NVME_REG_NSSR:
+		fallthrough;
+	case NVME_REG_CRTO:
+		return true;
+	default:
+		break;
+	}
+
+	return false;
+}
+
+static int get_register_property(int fd, void **pbar, int offset)
+{
+	int err, size = getpagesize();
+	__u64 value;
+	void *bar = malloc(size);
+	struct nvme_get_property_args args = {
+		.args_size	= sizeof(args),
+		.fd		= fd,
+		.offset		= offset,
+		.value		= &value,
+		.timeout	= NVME_DEFAULT_IOCTL_TIMEOUT,
+	};
+
+	if (!bar) {
+		nvme_show_error("malloc: %s", strerror(errno));
+		return -1;
+	}
+
+	memset(bar, 0xff, size);
+
+	err = nvme_get_property(&args);
+	if (nvme_status_equals(err, NVME_STATUS_TYPE_NVME, NVME_SC_INVALID_FIELD)) {
+		err = 0;
+		value = -1;
+	} else if (err) {
+		nvme_show_error("get-property: %s", nvme_strerror(errno));
+	}
```

Comment:
`value` is left uninitialized in this case

Reply:
Fixed by the commit 8452df16.